lockbox-ios/Presenter/ItemListPresenter.swift

```diff
@@ -216,7 +224,7 @@ class ItemListPresenter {
 
         self.view?.bind(sortingButtonTitle: itemSortTextDriver)
 
-        self.itemListDisplayActionHandler.invoke(ItemListSortingAction.alphabetically)
+        self.itemListDisplayActionHandler.invoke(Constant.setting.defaultItemListSort.asAction())
```

Comment:
I think this means that the setting will be cleared / changed to the default if you hard-quit the app...